pkg/volume/util/atomic_writer_test.go

```diff
@@ -30,22 +30,17 @@ import (
 	"testing"
 
 	"k8s.io/apimachinery/pkg/util/sets"
-	utiltesting "k8s.io/client-go/util/testing"
 )
 
 func TestNewAtomicWriter(t *testing.T) {
-	targetDir, err := utiltesting.MkTmpdir("atomic-write")
-	if err != nil {
-		t.Fatalf("unexpected error creating tmp dir: %v", err)
-	}
-	defer os.RemoveAll(targetDir)
+	targetDir := t.TempDir()
 
 	_, err = NewAtomicWriter(targetDir, "-test-")
 	if err != nil {
 		t.Fatalf("unexpected error creating writer for existing target dir: %v", err)
 	}
 
-	nonExistentDir, err := utiltesting.MkTmpdir("atomic-write")
+	nonExistentDir, err := os.MkdirTemp("", "atomic-write")
```

Comment:
Why not to use `t.TempDir()` here as well?

Reply:
done